src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -421,6 +481,49 @@ const SearchFieldList = ({ onSearch }: Props) => {
                     select
                     disabled={!searchParams.includeStudent}
                   />
+                  {switchYearRange == true ? (
+                    <SearchField
+                      name="graduation_year"
+                      value={searchParams.graduation_year}
+                      updateValue={handleUpdate}
+                      options={Array.from({ length: userProvidedYear - 1889 + 1 }, (_, i) => ({
+                        value: (userProvidedYear - i).toString(),
+                        label: (userProvidedYear - i).toString(),
+                      }))}
+                      Icon={FaCalendarTimes}
+                      disabled={!searchParams.includeAlumni}
+                      select
+                    />
+                  ) : (
+                    <Grid item width={225}>
+                      <Slider
+                        getAriaLabel={() => 'graduationYearRange'}
+                        value={graduationYearRange}
+                        onChange={handleSliderChange}
+                        valueLabelDisplay="auto"
+                        getAriaValueText={valuetext}
+                        min={1889}
+                        max={userProvidedYear}
+                        disabled={!searchParams.includeAlumni}
+                      />
+                      <Typography fontSize={15} align="center">
+                        {graduationYearRange[0]}-{graduationYearRange[1]}
+                      </Typography>
+                    </Grid>
+                  )}
+                  {switchYearRange == true ? (
+                    <FormControlLabel
+                      control={<Switch onChange={handleSwitchChange}></Switch>}
+                      label="Search by Year Range"
+                      labelPlacement="end"
+                    ></FormControlLabel>
+                  ) : (
+                    <FormControlLabel
+                      control={<Switch onChange={handleSwitchChange}></Switch>}
+                      label="Search by Graduation Year"
+                      labelPlacement="end"
+                    ></FormControlLabel>
+                  )}
```

Comment:
We can condense this conditional case - since the FormControlLabel is always rendered and it's content is almost exactly the same, except the label:
```suggestion
                 <FormControlLabel
                      control={<Switch onChange={handleSwitchChange} />}
                      label={switchYearRange ? "Search by Year Range" : "Search by Graduation Year"}
                      labelPlacement="end"
                   />
```

Reply:
That is true, it would make it more concise. Just changed it, thank you!